pkg/resources/machine/common_test.go

```diff
@@ -21,8 +21,11 @@ import (
 	"reflect"
 	"testing"
 
+	anexia "github.com/kubermatic/machine-controller/pkg/cloudprovider/provider/anexia"
+	anexiaTypes "github.com/kubermatic/machine-controller/pkg/cloudprovider/provider/anexia/types"
```

Comment:
Please avoid uppercases in package aliases.

Reply:
just all lowercase then or something else preferred?

Reply:
pushed as all lowercase now, will change if you give another preferred way

out of curiosity: do you happen to have anything to read for me why to avoid that? :)